content/docs/schema.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Anatomy of the Tina schema
+---
+
+The Tina schema is a way of defining the shape of your content for your entire site.  With traditional content management systems you may have done this sort of content modeling via GUI, however, given its tight coupling to Git, TinaCMS considers the filesystem the ultimate source of truth and leverages a "content-modeling as code" approach. 
+
+To that end, your schema is defined in a file called `.tina/schema.ts` (only `.ts` is supported for now).
+
+```ts
+// .tina/schema.ts
+import { defineSchema } from '@tinacms/cli'
+
+export default defineSchema({
+  collections: [
+    {
+      label: 'Blog Posts',
+      name: 'post',
+      path: 'content/posts',
+      fields: [
+        {
+          type: 'text',
+          label: 'Title',
+          name: 'title',
+        },
+        {
+          type: 'reference',
+          label: 'Author',
+          name: 'author',
+          collection: 'authors',
+        },
+      ],
+    },
+    {
+      label: 'Authors',
+      name: 'authors',
+      path: 'content/authors',
+      fields: [
+        {
+          type: 'text',
+          label: 'Name',
+          name: 'name',
+        },
+        {
+          type: 'text',
+          label: 'Avatar',
+          name: 'avatar',
+        },
+      ],
+    },
+  ],
+})
+```
+
+# `collections`
+
+The top-level key in the schema is an array of _collections_, a `collection` informs the API about _where_ to save content. You can see from the example that a `posts` document would be stored in `content/posts`. You can supply either `fields` or `templates` to define shape of your collection. For the most part you'll likely just want to use `fields`
```

Comment:
what does it mean for the most part, when would I want to use templates instead of fields?

Reply:
templates could be use when you have different types of the same thing. Kinda like [Polymorphism](https://en.wikipedia.org/wiki/Polymorphism_(computer_science)) If you are familiar with that. One example might be you have Feature pages, Content pages, and Blog pages but you want them all to be under the collection page. In this example all the different types of pages (content, feature, blog) would have different fields.

Reply:
So if all your collection documents share the same fields, just use `fields`, i
if you need to use different templates inside a collection (e.g about page, pricing page, jobs page inside a `pages`collections) then use `templates`.

Seen no need to use fancy words to explain that 😉